### ISSUE TITLE
CPDLC: Connection remains active if station goes offline

### ISSUE THREAD

**Issue by BravoMike99** [flybywiresim/a32nx#8038]
### Aircraft Version

Experimental

### Build info

```json
{
    "built": "2023-05-17T19:05:00+00:00",
    "ref": "refs/heads/experimental",
    "sha": "e2da961b511bf8d5c29524ff135cfbe41aa2d83f",
    "actor": "beheh",
    "event_name": "manual",
    "pretty_release_name": "experimental:e2da961b",
    "version": "v0.10.0-exp.e2da961"
}
```

### Describe the bug

If a hoppie station disconnects/goes offline without performing a LOGOFF request while a CPDLC connection is active, the connection never drops unless a manual disconnect is enforced through the CONNECTION STATUS page.

### Expected behavior

CPDLC connection to drop after a while

### Steps to reproduce

1) Notify and logon to a hoppie station
2) Disconnect the station from the hoppie network/close the client

### Discord Username (optional)

Bruno_pt99#5802

**Comment by svengcz** [comment]
I don't think this is a bug. That's what I would expect. Theoretically is it possible to track the online stations to auto-logoff.
